src/python/grpcio_test/grpc_test/framework/face/testing/future_invocation_asynchronous_event_service_test_case.py

```diff
@@ -306,9 +308,10 @@ def testParallelInvocations(self):
         second_request = test_messages.request()
 
         first_response_future = self.stub.future_value_in_value_out(
-            name, first_request, _TIMEOUT)
+            name, first_request, test_constants.SHORT_TIMEOUT)
+        # TODO(bug 2039): use SHORT_TIMEOUT instead
```

Comment:
Every timeout in every test should be LONG_TIMEOUT unless the run-time of the test is necessarily the duration of the timeout (because the test is a test of behavior in the event of expiration). This test method isn't testing expiration, so I believe it should use LONG_TIMEOUT. Unless I'm mistaken about something, or the test fails?

Yes, this will change radically change the values used in this test instead of keeping them approximately the same, but the movement should be toward increasing stability in every dimension of change. Again, unless I'm mistaken or we have an as-yet-unknown-but-2039-type bug.